dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -1706,6 +1706,14 @@ public PhaseOneResult process(CompilationUnitTree root, UnderlyingAST underlying
          */
         public void handleArtificialTree(Tree tree) {}
 
+        /**
+         * Perform any actions required when CFG translation created a Tree that is not part of the
+         * original AST and added the TreePath.
```

Comment:
The `and added the TreePath` isn't clear at all. That is something that the translation already does? Or that is something that this method will do?
Making the distinction between the two versions clearer, and linking them with `@see` would be helpful.